Zend/tests/attributes/003_ast_nodes.phpt

```diff
@@ -0,0 +1,68 @@
+--TEST--
+Attributes can deal with AST nodes.
+--FILE--
+<?php
+
+define('V1', strtoupper(php_sapi_name()));
```

Comment:
This constant and the resulting code below is very difficult to read: could we use something like `'a custom defined version'` or anything that isn't hard to fish out below?

Reply:
The constant could be renamed and the value changed. The important thing is that it has to be a value that is not known at compile time to prevent the compiler from inlining it. The test case is primarily about testing values that have to be evaluated at runtime (AST values).